### PR TITLE
Support Cisco IOS-XR netflow v9 mpls ipv4-ipv6-field mode

### PR DESCRIPTION
**Pull request by junpei-yoshino**
In IOS-XR with below config, there are some error logs.
```
flow monitor-map hoge
   record mpls ipv4-ipv6-fields labels 3
```

There are 2 reasons.
[1] unsupport fields
- 84 Sampler Name
- 91 MPLS Prefix Length
- 234 Ingress VRF id
- 235 Egress VRF id

[2] scope field is not supported

### CODE REVIEW

File: lib/fluent/plugin/parser_netflow.rb [hunk 1]
```diff
@@ -25,7 +25,7 @@ def configure(conf)
 
         @templates = Vash.new()
         # Path to default Netflow v9 field definitions
-        filename = File.expand_path('../netflow.yaml', __FILE__)
+        filename = File.expand_path('../netflow_option_fields.yaml', __FILE__)
```
Comment (repeatedly) [inline lib/fluent/plugin/parser_netflow.rb]: File rename is needed?

File: lib/fluent/plugin/parser_netflow.rb [hunk 2]
```diff
@@ -42,6 +42,14 @@ def configure(conf)
             raise "Bad syntax in definitions file #{@definitions}"
           end
         end
+        # Path to default Netflow v9 scope field definitions
+        filename = File.expand_path('../netflow_scope_fields.yaml', __FILE__)
+
+        begin
+          @scopefields = YAML.load_file(filename)
```
Comment (repeatedly) [inline lib/fluent/plugin/parser_netflow.rb]: `@scope_fields` is better than `@scopefields`

**Comment by repeatedly** [comment]
Commented. Patch itself is good :+1:

File: lib/fluent/plugin/parser_netflow.rb (hunk 1 above)
Comment (junpei-yoshino) [inline lib/fluent/plugin/parser_netflow.rb]: I added netflow_scope_fields.yaml.
For better understanding, I prefer to rename this file.
It's not must.

**Comment by junpei-yoshino** [comment]
Thank you for your review.
I renamed "scopefields" to "scope_fields".

**Comment by junpei-yoshino** [comment]
I'm testing now.
In my enviroment, Cisco router seems to not match rfc 3954.
Please wait a merge for fix.

**Comment by junpei-yoshino** [comment]
I modified option field definition yaml file.
There are some difference from rfc3954.
I rewrite field type 31 and 48.

I think that is not best way.
Netflow exporter's implementation will be keep changing.
I recommend to define ":uint" symbol.
It's variable length unsigned integer.
length is defined by template.

If we reach a consensus, I replace here.
https://github.com/repeatedly/fluent-plugin-netflow/blob/master/lib/fluent/plugin/parser_netflow.rb#L212-L223

**Comment by junpei-yoshino** [comment]
I'll make new PullRequest about using :uint symbol.
This branch works well.

**Comment by repeatedly** [comment]
Thanks for hard work!
Just merged this patch.

**Comment by junpei-yoshino** [comment]
Thank you for merge!

**Pull request merged**